public/app/features/transformers/FilterByValueTransformer/ValueMatchers/BasicMatcherEditor.tsx

```diff
@@ -37,16 +46,42 @@ export function basicMatcherEditor<T = any>(
       [options, onChange, isInvalid, field, converter]
     );
 
-    return (
-      <Input
-        className="flex-grow-1"
-        invalid={isInvalid}
-        defaultValue={String(options.value)}
-        placeholder="Value"
-        onChange={onChangeValue}
-        onBlur={onChangeOptions}
-      />
+    const onChangeVariableValue = useCallback(
+      (value: string) => {
+        setInvalid(!validator(value));
+        onChange({
+          ...options,
+          value: value,
+        });
+      },
+      [setInvalid, validator, onChange, options]
     );
+
+    //TODO: make regex matcher use a simple input, witaout suggestions.
```

Comment:
```suggestion
    // TODO: make regex matcher use a simple input, without suggestions.
```

Reply:
Thanks, but seeing this I realized this todo has already been done 🤡